config.kiwi

```diff
@@ -23,19 +23,16 @@
   <preferences profiles="wsl">
     <type image="tbz" />
   </preferences>
-  <repository type="rpm-md" imageinclude="true" alias="tumbleweed">
-    <!--
-      Use factory, as Leap 15.6 systemd is too old to work correctly on WSL
-      (We require at least 255.7); see:
-      https://github.com/systemd/systemd/pull/32534
-    -->
-    <source path="obs://openSUSE:Factory/standard" />
+  <repository type="rpm-md" imageinclude="true" alias="leap-16.0-oss">
+    <!-- Primary openSUSE Leap 16.0 repository -->
+    <source path="https://download.opensuse.org/distribution/leap/16.0/repo/oss/" />
   </repository>
-  <repository type="rpm-md" imageinclude="true" alias="tumbleweed-aarch64" profiles="lima">
-    <!-- The lima image may need aarch64 ports (for mac M1) -->
-    <source path="http://download.opensuse.org/ports/aarch64/tumbleweed/repo/oss/" />
+  <repository type="rpm-md" imageinclude="true" alias="leap-16.0-ports-aarch64" profiles="lima">
+    <!-- aarch64 for lima (if you need ports) -->
+    <source path="https://download.opensuse.org/ports/aarch64/leap/16.0/repo/oss/" />
```

Comment:
This URL is a 404. I think that's because Leap 16 actually just supports aarch64 directly, without needing ports (so this whole `<repository>` can be removed).
See:
- https://download.opensuse.org/distribution/leap/16.0/repo/oss/aarch64/
- https://download.opensuse.org/distribution/leap/16.0/offline/